src/modules/client/components/ClientAlerts.tsx

```diff
@@ -0,0 +1,150 @@
+      body: `${theme.palette.error.main}04`,
+      border: `${theme.palette.error.main}50`,
+    },
+    medium: {
+      header: theme.palette.alerts.mediumPriorityHeaderBackground,
+      body: theme.palette.alerts.mediumPriorityBodyBackground,
+      border: theme.palette.alerts.mediumPriorityBorder,
+    },
+    low: {
+      header: theme.palette.alerts.lowPriorityHeaderBackground,
+      body: theme.palette.alerts.lowPriorityBodyBackground,
+      border: theme.palette.alerts.lowPriorityHeaderBackground,
+    },
+  };
+
+  return (
+    <Alert
+      sx={{
+        p: 0,
+        color: 'black',
+        backgroundColor: priorityColors[priority].body,
+        borderColor: priorityColors[priority].border,
+        '& .MuiAlert-message': {
+          p: 0,
+          width: '100%',
+        },
+      }}
+      icon={false}
+      variant='outlined'
+    >
+      <AlertTitle
+        sx={{
+          p: 2,
+          m: 0,
+          backgroundColor: priorityColors[priority].header,
+          fontWeight: 'bold',
+          textTransform: 'capitalize',
+        }}
+      >
+        {priority} Priority Alert
+      </AlertTitle>
+      <Box sx={{ p: 2 }}>
+        {shouldShowClientName && (
+          <Typography variant='body2' sx={{ pb: 1 }}>
+            {clientNameAllParts(client)}
+          </Typography>
+        )}
+        <Typography variant='body1' sx={{ pb: 1 }}>
+          {alert.note}
+        </Typography>
+        <Typography variant='body2' sx={{ fontSize: '12px' }}>
+          Created by {alert.createdBy?.name} on{' '}
+          {parseAndFormatDateTime(alert.createdAt)}.
+          {alert.expirationDate &&
+            ` Expires on ${parseAndFormatDateTime(alert.expirationDate)}.`}
+        </Typography>
+      </Box>
+    </Alert>
+  );
+};
+
+export enum AlertContext {
+  Client = 'Client',
+  Household = 'Household',
+}
+
+interface ClientAlertsProps {
+  clients: ClientFieldsFragment[] | HouseholdClientFieldsClientFragment[];
+  alertContext: AlertContext;
+  loading?: boolean;
+}
+
+const ClientAlerts: React.FC<ClientAlertsProps> = ({
+  clients,
+  alertContext = AlertContext.Client,
+  loading = true,
+}) => {
+  const clientAlerts: ClientAlertCardProps[] = [];
+  if (!clients) return;
+  clients.forEach((c) => {
+    if (c.access.canViewClientAlerts) {
+      c.alerts.forEach((a) => {
+        clientAlerts.push({
+          alert: a,
+          client: c,
+          shouldShowClientName: alertContext === AlertContext.Household,
+        });
+      });
+    }
+  });
+
+  const title = `${alertContext} Alerts (${clientAlerts.length})`;
+  if (loading && clients.length === 0) return <Loading />;
+
+  return (
+    <TitleCard
+      title={title}
+      headerVariant='border'
+      headerTypographyVariant='body1'
+    >
+      <Box sx={{ m: 2 }}>
+        {clientAlerts.length === 0 &&
+          `${alertContext} has no alerts at this time`}
+        <Stack gap={2}>
+          {clientAlerts.map((ca) => (
```

Comment:
Should alerts perhaps be ordered by their priority, then by date?

Reply:
Yeah that probably makes sense. Thanks!

Reply:
If the list of alerts is shown in a context where multiple household members' alerts are present, should they be also ordered by household member? So all the alerts for Person A would appear first (ordered internally by priority then date), then all the alerts for Person B? Or should the whole list of alerts just be ordered by priority + date regardless of the clients they're associated with?

Reply:
resolved at standup! We agreed that high priority alerts should appear highest -- regardless if whether it's a multi-client household and different client alerts get jumbled together. Let me know if you agree that it then makes sense to sort the alerts here, rather than in the backend (which I think would require restructuring our graphql queries somewhat)